.gitpod/Dockerfile

```diff
@@ -1,7 +1,3 @@
-FROM gitpod/workspace-full:latest
```

Comment:
Please add this line back

Reply:
Yes , even I ran without this and faced some error to load with default image only , adding it back .